frontend/src/static/js/v_summary.js

```diff
@@ -863,6 +951,14 @@ window.vSummary = {
       info.fileTypeColors = this.fileTypeColors;
     });
   },
+  mounted() {
+    this.resetMergedGroupsMap();
+
+    // restoring custom merged groups after watchers finish their job
+    setTimeout(() => {
```

Comment:
Would setting the timeout to be 0ms work as well? Since the point is to push the `restoreMergedGroups()` to the end of the execution queue. 

Currently, when you try to reload a page with some groups merged, there seems to be a slight delay on the page before they merge.

Reply:
Thanks for the advice. But I changed the timeout to 0 but the delay still seems to be there.